contributing/style_guide.md

```diff
@@ -354,15 +354,15 @@ export function foo(): string {
 
 `https://deno.land/std/` is intended to be baseline functionality that all Deno
 programs can rely on. We want to guarantee to users that this code does not
-include potentially unreviewed third party code.
+include potentially unreviewed third-party code.
 
 #### Document and maintain browser compatibility.
 
-If a module is browser compatible, include the following in the JSDoc at the top
+If a module is browser-compatible, include the following in the JSDoc at the top
```

Comment:
I'd appreciate if you could do the same change in deno_std.

https://github.com/denoland/deno_std/search?q=browser+compatible

Reply:
On it 😄